### PR TITLE
Made changes to the header

### PR DESCRIPTION
**Pull request by Mr-DanielDang**
Here are the changes to the header.
I need some help with the alignment on the list elements.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,22 +10,23 @@
...
-            <div class="header-container">
+        <header id="header-bg">
+            <div class="header-container" id="header">
                 <nav class="navigation">
                     <ul>
-                        <li><a><img src="images/image_logo.png">Friday</a></li>
-                        <li><a>About</a></li>
-                        <li><a>Contact</a></li>
-                        <li><a><img src="#"></a></li>
+                        <!-- Need help with alignment, please. -->
```
Comment (jerrylammel) [inline index.html]: hi Daniel, you could set the ul as a flex by display: flex, so as to align them according to screen sizes

File: index.html [hunk 2]
```diff
@@ -10,22 +10,23 @@
...
+                        <li><a><img src="images/image_logo.png" alt="Friday Logo" id="logo"></a></li>
```
Comment (jerrylammel) [inline index.html]: basically, we should have all resources such as images, other css files managed under the folder resources. Could you move that?

File: index.html [hunk 3]
```diff
@@ -10,22 +10,23 @@
...
+                        <li><a><img src="images/instagram_logo.png" alt="Contact Icon" id="contact"></a></li>
                     </ul>
                 </nav>
             </div>
         </header>
         <main> 
             <section class="banner">
                 <h2>Friday, plan your next week</h2>
-                <div class="background image"><img src="images/image_banner.jpg" alt="Lady smiling happily on street"></div>
+
```
Comment (jerrylammel) [inline index.html]: hi Daniel, it sounds that you overwrote this change for the banner image from Sam. Please get pull the latest commits from main repository and merge yours

File: index.html [hunk 4]
```diff
@@ -10,22 +10,23 @@
         <title>Friday</title>
     </head>
     <body>
-        <header>
-            <div class="header-container">
+        <header id="header-bg">
```
Comment (jerrylammel) [inline index.html]: I would suggest set the background to the first div under header

File: style.css [hunk 5]
```diff
@@ -12,6 +12,37 @@ html {
     text-align: center; 
 }
 
+#header-bg {
+    background-color: var(--background-color-header);
+    height: 80px;
+}
+
+#header {
+    margin: 0em 5em;
```
Comment (jerrylammel) [inline style.css:43]: could you use rem rather than em for better alignment and scaling to the rest of elements?

File: style.css [hunk 6]
```diff
@@ -12,6 +12,37 @@ html {
...
+    min-width: auto;
+    max-height: auto;
```
Comment (jerrylammel) [inline style.css:45]: we should make a nav bar fixed at the top so when users scrolling down our page the nav bar would be still there. That said, you need to set position as fixed and set a height for it

File: style.css [hunk 7]
```diff
@@ -12,6 +12,37 @@ html {
...
+}
+
+/* NEED HELP HOW TO ALGIN LIST
+    Mr-DanielDang */
+#header ul {
+    list-style: none;
+}
+
+#header li {
```
Comment (jerrylammel) [inline style.css]: could you explain why you need to set li elment to display inline?

**Review by jerrylammel — CHANGES_REQUESTED** [review]

File: style.css (hunk 7 above)
Comment (Mr-DanielDang) [reply style.css]: To display the list horizontally

**Comment by jerrylammel** [comment]
ok, thanks

**Comment by jerrylammel** [comment]
this ok

File: style.css (hunk 5 above)
Comment (jerrylammel) [reply style.css:43]: please be sure to use rem in later version

**Pull request merged**